src/docfx/lib/schema/JsonSchemaValidator.cs

```diff
@@ -578,6 +584,131 @@ private static void ValidateDeprecated(JsonSchema schema, string propertyPath, J
             }
         }
 
+        private void ValidateAnyOf(JsonSchema schema, string propertyPath, JToken token, List<Error> errors)
+        {
+            if (schema.AnyOf.Length <= 0)
```

Comment:
nit: when would this be `<`?

Reply:
all the other checks uses `<=`, same here :)